database/seeds/UsersTableSeeder.php

```diff
@@ -66,6 +66,7 @@ private function createAdmin()
         }
         $user->roles()->attach(Role::getId(Role::PRINT_ADMIN));
         $user->roles()->attach(Role::getId(Role::INTERNET_ADMIN));
+        $user->roles()->attach(Role::getId(Role::LOCALE_ADMIN));
```

Comment:
This is enough to get this feature out. Later we should use the object field for this role so we can assign this privilege for each language separately.